src/sentry/workflow_engine/migration_helpers/rule.py

```diff
@@ -110,3 +115,110 @@ def create_if_dcg(
 def create_workflow_actions(if_dcg: DataConditionGroup, actions: list[dict[str, Any]]):
     # TODO: create actions, need registry
     pass
+
+
+def update_migrated_issue_alert(rule: Rule, data: dict[str, Any], user: RpcUser | None = None):
```

Comment:
is there a way to ensure we have everything we need in `data`, like the keys we expect to exist actually exist? a `TypedDict` or a `dataclass` could be a good way to do this, that way we don't have to do `data.get()`

Reply:
I'll add a `TypedDict`, but some of the fields are optional so they would still need `data.get()`

Reply:
you can make the type `None`?